### DISCUSSION TITLE
I want to seperate credential and google depend on pages

### DISCUSSION THREAD

**Discussion by yusa-imit** [nextauthjs/next-auth#3277] [Help] [answered]
My site was not needed to be logged in before for normal visitors.
But, with some feature changes, I have to implement a login feature for visitors.

So, I'm currently using just Credential for my ```/admin``` page. I don't wanna see Google login on this page.
And I want Visitors can log in only with Google login provider in page like ```/ ``` or ```/blahblah```, not Credential.

Is this possible?

**Comment by balazsorban44** [comment, marked as answer]
https://next-auth.js.org/configuration/initialization#advanced-initialization

**Reply by yusa-imit** [reply]
Oh, I missed that advanced section.
Thank you for answering this dumb question.
